app/src/main/java/com/flatstack/android/DI.java

```diff
@@ -25,38 +22,22 @@ public static File getCacheDir() {
             : App.appContext.getCacheDir();
     }
 
-    private static volatile OkHttpClient okHttpInstance = null;
-
-    public static OkHttpClient getHttpClient() {
-        OkHttpClient localInstance = okHttpInstance;
-        if (localInstance == null) {
-            synchronized (OkHttpClient.class) {
-                localInstance = okHttpInstance;
-                if (localInstance == null) {
-                    okHttpInstance = localInstance = new OkHttpClient.Builder()
-                        .cache(new okhttp3.Cache(getCacheDir(), 20 * 1024 * 1024))
-                        .build();
-                }
-            }
-        }
-        return localInstance;
-    }
+    private static volatile Glide glideInstance = null;
 
-    private static volatile Picasso picassoInstance = null;
-
-    public static Picasso getPicasso() {
-        Picasso localInstance = picassoInstance;
-        if (localInstance == null) {
-            synchronized (Picasso.class) {
-                localInstance = picassoInstance;
-                if (localInstance == null) {
-                    picassoInstance = localInstance = new Picasso.Builder(App.appContext)
-                        .downloader(new OkHttp3Downloader(getHttpClient()))
-                        .build();
-                }
-            }
-        }
-        return localInstance;
+    public static Glide getImageLoader() {
+        return Glide.get(App.appContext);
```

Comment:
there's not need for that, Glide.with does not hold the reference to an `Activity`, it's smarter than that :)